gateway/fedimint-lightning/src/lnd.rs

```diff
@@ -1375,8 +1377,10 @@ impl ILnRpcClient for GatewayLndClient {
         let pending_inbound = channel_balance_response
             .pending_open_remote_balance
             .unwrap_or_default();
-        let inbound_lightning_liquidity_msats =
-            total_inbound.msat - unsettled_inbound.msat - pending_inbound.msat;
+        let inbound_lightning_liquidity_msats = total_inbound
+            .msat
+            .saturating_sub(unsettled_inbound.msat)
+            .saturating_sub(pending_inbound.msat);
```

Comment:
Is saturation the right semantic here? Why would it go negative in the first place?

Reply:
I think so but I'm open to being convinced otherwise. I'm not quite sure why it would be negative, the only explanation is that there is double counting of sats between `unsettled_inbound` and `pending_inbound` by LND. 

I think saturation is fine since this is just showing the user their inbound balance, so it will just show 0, which seems like a reasonable thing to do.